core/src/test/java/com/linecorp/armeria/server/ParameterizedPathMappingTest.java

```diff
@@ -161,4 +160,33 @@ void patternString() {
         assertThat(pathMappingWithComplexPattern.patternString())
                 .isEqualTo("/service/:value/items/:value/:itemId");
     }
+
+    @Test
+    void captureTheRestPattern() {
+        final ParameterizedPathMapping ppm = new ParameterizedPathMapping("/service/{*value}");
+        final RoutingResult matchSingleParam = ppm.apply(create("/service/hello", "foo=bar")).build();
+        assertThat(matchSingleParam.isPresent()).isTrue();
+        assertThat(matchSingleParam.pathParams()).containsEntry("value", "hello");
+
+        final RoutingResult matchMultiParams = ppm.apply(create("/service/foo/bar", "foo=bar")).build();
+        assertThat(matchMultiParams.isPresent()).isTrue();
+        assertThat(matchMultiParams.pathParams()).containsEntry("value", "foo/bar");
+
+        final ParameterizedPathMapping emptyParam = new ParameterizedPathMapping("/service/{*}");
```

Comment:
I'm not sure this is a valid one or not. 😅 
Is there any reason that you define it as a valid one? Or, do you have any reference to this pattern?

Reply:
> Is there any reason that you define it as a valid one?

Yes. I tried testing with spring [PathPatternParser](https://github.com/spring-projects/spring-framework/blob/main/spring-web/src/main/java/org/springframework/web/util/pattern/PathPatternParser.java).
`PathPatternParser` seems to allow `{*}` so I added this test case to match the result. 
But it might be a bug. 😅

Reply:
> But it might be a bug of spring. 😅

I believe so. 😅  Let's raise an exception then. 😄